modules/ztp-sno-accelerated-ztp.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-accelerated-ztp_{context}"]
+= Accelerated provisioning of {ztp}
+
+:FeatureName: Accelerated provisioning of {ztp}
+include::snippets/technology-preview.adoc[]
+
+You can reduce the time taken for installation by using accelerated provisioning of {ztp} for {sno}. 
```

Comment:
Suggestion to add "cluster" here for extra clarity, but up to you:
```suggestion
You can reduce the time taken for cluster installation by using accelerated provisioning of {ztp} for {sno}. 
```

Reply:
thanks for the review @skopacz1 

Yes, makes sense.